pkg/nvlib/device/device.go

```diff
@@ -208,6 +209,33 @@ func (d *device) IsMigEnabled() (bool, error) {
 	return (mode == nvml.DEVICE_MIG_ENABLE), nil
 }
 
+// IsImexEnabled checks if a device has IMEX capabilities.
+func (d *device) IsImexEnabled() (bool, error) {
```

Comment:
I'm fine with using IMEX in the name of the abstractions we create in GFD, but we shouldn't tie the name of this function to the fact that we use it for IMEX related things in GFD. It seems more general than that.

Reply:
We are checking for the `GpuFabric` symbol, so something around that? `HasGpuFabricCapabilities` or a similar name

Reply:
Yeah, let's call is `HasGpuFabricInfo()`

Reply:
renamed